### PR TITLE
[IMP] Cash_moves : clean pop-up and allow it at closing

### PR DESCRIPTION
**Pull request by mobt-odoo**
Description of the issue/feature this PR addresses: 

Improve UI and UX

Current behavior before PR:

Desired behavior after PR is merged:

- make 'reason' required
- remove amount on confirm button
- bring method at closing

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by vlst-odoo — CHANGES_REQUESTED** [review]
don't forget to add a commit message

File: addons/point_of_sale/static/tests/tours/chrome_tour.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ registry.category("web_tour.tours").add("ChromeTour", {
             Dialog.confirm("Open Register"),
             Chrome.clickMenuButton(),
             Chrome.clickMenuDropdownOption("Cash In/Out"),
+            Chrome.fillTextArea(".cash-reason", "MOBT"),
```
Comment (caburj) [inline addons/point_of_sale/static/tests/tours/chrome_tour.js:17]: you even have a test. XD

**Review by caburj — APPROVED** [review]
needs rebase before it can be merged.

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**